cobalt/browser/cpu_usage_tracker.cc

```diff
@@ -251,6 +262,37 @@ void CpuUsageTracker::StopAndCaptureOneTimeTracking() {
   }
 }
 
+void CpuUsageTracker::UpdateIntervalsEnabled(bool enabled) {
+  if (intervals_enabled_ == enabled) return;
+  if (!task_runner_->RunsTasksInCurrentSequence()) {
+    task_runner_->PostTask(
+        FROM_HERE, base::BindOnce(&CpuUsageTracker::UpdateIntervalsEnabled,
+                                  base::Unretained(this), enabled));
+    return;
+  }
+  base::AutoLock auto_lock(lock_);
+  intervals_enabled_ = enabled;
+  storage_->Set(kIntervalsEnabled, base::Value(intervals_enabled_));
+  for (auto& pair : interval_contexts_) {
```

Comment:
Each pair is an IntervalContext, right? Could we make the type an IntervalContext or call the variable something like interval_context/interval_context_pair to make it clear when reading it?

Reply:
Yeah looks good, thanks